scripts/git/fix_copybara_merge.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Fixes a Copybara push that failed to create a merge commit, using the
+# COPYBARA_TAG label to add a second parent to the HEAD commit.
+# This is mostly intended to be used by automation.
```

Comment:
Could expand on this with:

* when this needs to be run
* the branches this changes (`google` branch only? it looks like this can run on any branch... but we should guard against changing `main`)
* a TODO to fix the underlying Copybara bug(s) and remove this script?
* how to know that this was successful (could talk about "1 commit ahead, 0 commits behind" vs "50 commits ahead, 50 commits behind")
* what to do after running the script (verify that it was successful by looking at the log, then force push over `google`?)

Reply:
Added more documentation and a check that the current branch is the google branch.

I have no real hope of Copybara being fixed and in fact think we should probably move to having it not even try to create merge commits, instead relying on our own GitHub action. That would allow us to make it fail migration on an integrate failure and make the path here consistent.